packages/server/src/handlers/PostHandler.ts

```diff
@@ -65,7 +65,7 @@ export class PostHandler extends BaseHandler {
 
     let id
     try {
-      id = this.options.namingFunction(req, metadata)
+      id = await Promise.resolve(this.options.namingFunction(req, metadata))
```

Comment:
you don't technically need the `Promise.resolve` wrapper, you can simply await it, makes it cleaner

Reply:
You are right of course, I did actually did not know that so now I have learned something as well, thanks! :)